lib/rpm_package.py

```diff
@@ -130,3 +130,11 @@ def _load(self):
                     distro_version=self.distro.version)
         except TypeError:
             raise exception.PackageDescriptorError(package=self.name)
+
+    @property
+    def version(self):
+        return self.spec_file.query_tag("version")
+
+    @property
+    def release(self):
+        return self.spec_file.query_tag("release").split('.')[0]
```

Comment:
Ok, returning the whole string since it will be used in create_release_notes.py.